Sources/Core/Payment/IBAN.swift

```diff
@@ -102,3 +147,79 @@ public enum IBAN {
         return check == 1
     }
 }
+
+public class IBANFormatter: Formatter {
+    public var placeholder: String {
+        didSet {
+            print("IBAN placeholder: \(placeholder)")
```

Comment:
Wollen wir ein Print in produktivem Code haben? Das kann ein Nutzer des SDKs nicht ausschalten.

Reply:
6f226c26780bb613c7d0dffd43dbe50c0539bc36